public/site.webmanifest

```diff
@@ -0,0 +1 @@
+{"name":"","short_name":"","icons":[{"src":"/android-chrome-192x192.png","sizes":"192x192","type":"image/png"},{"src":"/android-chrome-512x512.png","sizes":"512x512","type":"image/png"}],"theme_color":"#ffffff","background_color":"#ffffff","display":"standalone"}
```

Comment:
@kcoderhtml pls properly fill out meta here (name and such)

Reply:
ill work on this for you!
=

Reply:
Just fixed!